entity-framework/core/modeling/custom-function-mapping.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Custom function mapping - EF Core
+description: Mapping user-defined functions to database functions
+author: maumar
+ms.date: 11/23/2020
+uid: core/modeling/custom-function-mapping
+---
+# Custom function mapping
+
+EF Core allows for using user-defined SQL functions in queries. To do that, the functions need to be mapped to a CLR method during model configuration. When translating the LINQ query to SQL, user-defined function will be called instead of the CLR function it has been mapped to.
```

Comment:
"allows for using" did not parse properly in my english.
Also mention that the CLR method can be user defined or something which comes from other library.
Also mention that the function on server side can also be a built-in function rather than use defined function.